pkg/names/names.go

```diff
@@ -47,3 +47,7 @@ func ServiceMonitor(vcName string) string {
 func GrafanaDashboard(vcName string) string {
 	return vcName + "-grafana-dashboard"
 }
+
+func GrafanaDashboardJson(vcName string) string {
```

Comment:
I'd make it more clear in the name that it's the file's name. `GrafanaDashboardFile`. It's also not something that I see being used in different places, maybe it's even fine to have the name generation only in the reconcile function.

Reply:
Agree on the name change, the reason I put it in the names file is because I use it in both the reconcile loop and in test cases.